content/docs/command-reference/exp/run.md

```diff
@@ -118,6 +118,12 @@ changes between/after queueing runs.
 `dvc exp run --temp`, for example to continue working on the project meanwhile
 (e.g. on another terminal).
 
+> ⚠️ Note that only tracked code and data will be included in `--queue/temp`
```

Comment:
```suggestion
> ⚠️ Note that only tracked files and directories will be included in `--queue/temp`
```
Looks okay as is, but seems like `files and directories` is more general (covers params) and consistent with the rest of the paragraph.

Reply:
True. Committing. But then I think I'll change the next occurrence to just "files" (Git doesn't track dirs per-se anyway).